scrapy/utils/misc.py

```diff
@@ -184,6 +184,25 @@ def set_environ(**kwargs):
                 os.environ[k] = v
 
 
+def walk_callable(node):
+    """Similar to ``ast.walk``, but walks only function body and skips nested
+    functions defined within the node.
+    """
+    from collections import deque
```

Comment:
Could this import be moved to the top of the file?

Reply:
Done